source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -109,6 +109,8 @@ Removed features and their replacements
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
 | ``metacache_disable_merging``                              | Use :dovecot_plugin:ref:`metacache_index_merging=none <metacache_index_merging>` instead |
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| ``disable_plaintext_auth`` has been replaced with          | This has been replaced with :dovecot_core:ref:auth_allow_cleartext setting               |
```

Comment:
Im not sure I see the diff correctly in this UI, but it says "has been replaced with" twice. The sentence in the first column seems cut off.

Worlds smallest nit: Please add full stop at end of sentence. Could also do that for the row above.